src/tiny-inflate@1.0.3.esm.js

```diff
@@ -1,4 +1,6 @@
-/*
+/*!
```

Comment:
does `/*!` semantically mean something ? (just asking for ma curiosity)

Reply:
Yes, comment blocks starting with this will be considered license blocks and will not be removed from the minified output, so we are in accordance with the license terms of some third party code used (and also include our own license info in the minified dist files).

Reply:
good to know !